app/Http/Requests/CategoryRequest.php

```diff
@@ -26,7 +31,7 @@ public function authorize()
     public function rules()
     {
         return [
-            // 'name' => 'required|min:5|max:255'
+             'title' => 'required|min:3|max:255'
```

Comment:
use unique constrain

Reply:
when i set the title to be unique i can not edit the title again